src/routes/DiaryList.js

```diff
@@ -1,19 +1,43 @@
 import React from 'react'
 import { connect } from 'react-redux'
 import { Link } from 'react-router-dom'
+import Button from 'material-ui/Button'
+import AddIcon from '@material-ui/icons/Add'
+import Grid from 'material-ui/Grid'
+
+import { fetchDiaries } from '../services/localStorage'
 
 class DiaryList extends React.Component {
+  componentDidMount() {
+    this.props.dispatch({
+      type: 'LOAD_DIARIES',
+      diaries: JSON.parse(fetchDiaries())
```

Comment:
Ideally, this wouldn't have to do a JSON.parse.  `fetchDiaries` should return the JS object.

Reply:
makes sense